data/human/news.txt

```diff
@@ -238,6 +239,8 @@ news "company psychologist"
 			"My job is to help workers deal with stress and channel it in healthy directions. A little bit of stress is not a bad thing, if it drives someone to be more productive."
 			"We routinely evaluate employees for sociopathic tendencies, narcissism, and other signs that they might be excellent candidates for upper management positions."
 			"I've gotten incredibly good at doing this thing with my face that makes it look like I care."
+			"A trouble shared is a trouble halved, so most of the time I just need to pretend to listen to get some resource issue within acceptable limits."
```

Comment:
I can definitely tell the second half of this line needs some rewriting, but I don't know if I can suggest anything because I'm not too sure what it says myself. Specifically, if you could get rid of the repetition of `to` ("need _to_ pretend _to_ listen _to_") and reword `some resource issue within acceptable limits`, I think that would fix my concern with it.

Reply:
Yes, it didn't look nice. I've rewritten the sentence.

Is it clear from the sentence that the psychologist is referring to a person/patient when talking about `the resource`?

Reply:
Yep, thanks!